src/aas-services-support/ADT Support/AdtDtos.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
```

Comment:
Please don't overuse short names! I assume Dto means Data transfer object! Please use it that way